subprojects/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerExecutorParallelIntegrationTest.groovy

```diff
@@ -698,6 +700,12 @@ class WorkerExecutorParallelIntegrationTest extends AbstractWorkerExecutorIntegr
         succeeds("allTasks")
     }
 
+    @Requires(
+        value = IntegTestPreconditions.NotConfigCached,
+        reason = """Assumptions about project locking introduced in https://github.com/gradle/gradle/pull/2171 do not hold.
```

Comment:
Missed a reference to https://github.com/gradle/gradle/issues/25540  (which may be more relevant to https://github.com/gradle/gradle/pull/2171)

Reply:
Good catch, I'll update the link